celements-user-manager/web-module/src/main/webapp/templates/celAppScripts/cel/admin/UserAdmin.vm

```diff
@@ -0,0 +1,5 @@
+#if($isAdmin)
+<!-- UserAdmin already rendered with layout -->
+#else
+<p>You have no access rights for this page.</p>
```

Comment:
Can we use 'celInlineTemplates/accessdenied.vm' or parts of it here instead of this hard coded string?

Reply:
the else expression will only be executed in a error or hack case. That means I think it may look shabby.